corehq/apps/translations/integrations/transifex/client.py

```diff
@@ -216,7 +211,15 @@ def get_source_lang(self):
         """
         :return: source lang code on transifex
         """
-        return self.project_details().json().get('source_language_code')
+        source_language = self.project.related["source_language"]
+        return self._language_id_to_lang_code(source_language.id)
+
+    def get_project_langs(self):
```

Comment:
we can't get this from project details?

Reply:
I updated this from `get_project_details` since this was the only thing used out of the details. If you mean that this shouldn't have to use `ResourceLanguageStats` though, I think you are correct. It looks like I can directly access `self.project.languages` and append `self.project.source_language`. Will update.